NAS2D/Mixer/Mixer.h

```diff
@@ -41,121 +39,46 @@ namespace NAS2D
 		virtual ~Mixer() = default;
 
 	public:
-		/**
-		 * Plays a sound on the first available sound channel.
-		 *
-		 * \param	sound	A reference to a Sound Resource.
-		 */
 		virtual void playSound(const Sound& sound) = 0;
-
-		/**
-		 * Stops playing all sounds on all channels.
```

Comment:
I suppose the "all channels" part on some of those methods can be implied. That leaves the rest quite redundant.